program/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub mod instruction;
+pub mod state;
+
+solana_program::declare_id!("PStake1111111111111111111111111111111111111");
```

Comment:
???

Reply:
This is just a temporary id, not a real one.